### DISCUSSION TITLE
Did someone push a test email to production? (Password reset email)

### DISCUSSION THREAD

**Discussion by ghost** [elrumo/macOS_Big_Sur_icons_replacements#1228] [General]
I got a password reset email, and the website doesn't even have a password reset feature yet.

**Comment by elrumo** [comment]
No test emails have been sent recently, what's your email address or username? And the website has had a password reset feature since accounts were launched: 
![image](https://user-images.githubusercontent.com/36299870/123932352-45385a00-d989-11eb-97cf-bee90da366d7.png)

**Comment by ghost** [comment]
_(Oh, I didn't notice the Reset Password option before.)_
Since "change password" wasn't available in my account settings, I thought you haven't implemented it yet.

Sorry. My bad, I must have accidentally clicked on it then. Sorry for opening a new discussion.

**Suggestion:** Clicking on it should confirm with the user before sending the email. IMO.

**Reply by elrumo** [reply]
Good suggestions, thanks. I implemented that quickly and that small detail went over my head, I'll add a confirmation and add that to the user settings panel too. Thanks!
